README.md

```diff
@@ -1 +1,94 @@
-# pdv-ms-tokenizer 
+# PDV MS-Tokenizer
+
+Spring Application implementing the **Tokenizer** microservice for the **Personal Data Vault** Project.
+
+This microservice works in conjunction with:
+
+- [pdv-ms-person](https://github.com/pagopa/pdv-ms-person)
+- [pdv-ms-user-registry](https://github.com/pagopa/pdv-ms-user-registry)
```

Comment:
from the point of view of the _tokenizer_, it is not aware of the existence of other microservices in the same _ecosystem_. So, I suggest splitting dependencies into upstream and downstream
```suggestion
---

## Dependencies

Upstream:
- [pdv-ms-user-registry](https://github.com/pagopa/pdv-ms-user-registry)

Downstream:
- [Amazon DynamoDB](https://aws.amazon.com/dynamodb/?nc1=h_ls)
```

Reply:
in these lines I wanted to highlight that this micro-service is part of the Personal Data Vault project and to give a global point of view I added these two links to easily navigate them. I think that we could keep both the information.

```suggestion
This microservice is part of the Personal Data Vault Project and works in conjunction with:

- [pdv-ms-person](https://github.com/pagopa/pdv-ms-person)
- [pdv-ms-user-registry](https://github.com/pagopa/pdv-ms-user-registry)

## Dependencies
Upstream:
- [pdv-ms-user-registry](https://github.com/pagopa/pdv-ms-user-registry)
Downstream:
- [Amazon DynamoDB](https://aws.amazon.com/dynamodb/?nc1=h_ls)

```